src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java

```diff
@@ -138,4 +146,14 @@ private Participation getStudentParticipation(List<Participation> participationL
                             String.format(MESSAGE_INVALID_TUTORIAL_FOR_STUDENT, tutorial.getSubject()));
                 });
     }
+
+    private boolean containsAttendance(List<Attendance> attendanceList) {
+        assert attendanceList != null;
+        for (Attendance currentAttendance : attendanceList) {
+            if (currentAttendance.isSameWeek(attendance)) {
```

Comment:
Should check for exact date rather than week

Reply:
Yep changed, thanks!